service/debugger/debugger.go

```diff
@@ -546,7 +546,7 @@ func (d *Debugger) Restart(rerecord bool, pos string, resetArgs bool, newArgs []
 		return nil, fmt.Errorf("could not launch process: %s", err)
 	}
 
-	discarded := []api.DiscardedBreakpoint{}
```

Comment:
I don't like this level of style enforcement. This seems too arbitrary to me.

Reply:
That's fine, we don't have to enforce it going forward to new PRs or anything, it's just a bit of cleanup. Changes things somewhat semantically, but nothing that has any real effect here.